tests/apollo/test_skvbc_restart_recovery.py

```diff
@@ -364,6 +364,48 @@ async def test_recovering_fast_path(self, bft_network, skvbc, constant_load):
 
             bft_network.start_replica(non_primary)
 
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: c == 0, rotate_keys=True)
+    @verify_linearizability()
+    async def test_restart_f_replicas_verify_fastpath(self, bft_network, tracker):
+        """
+        1. Start all Replicas.
+        2. Chose F non Primary Replicas to stop.
+        3. Advance the others 5 Checkpoints.
+        4. Bring back the previously stopped F replicas.
+        5. Wait for State Transfer to finish on all.
+        6. Verify the system is processing on Fast Path.
+        7. Goto Step 2.
+        """
+        # start replicas
+        [bft_network.start_replica(i) for i in bft_network.all_replicas()]
+
+        #log = foo()
+        loop_counter = 0
+        while (loop_counter < 1):
```

Comment:
isn't this supposed to run for more than 1 iteration?

Reply:
Done